lib/legacy/class.html

```diff
@@ -96,15 +95,26 @@
     // If lifecycle is called (super then me), order is
     // (1) C.created, (2) A.created, (3) B.created, (4) element.created
     // (again same as 1.x)
-    function _mixinBehaviors(behaviors, klass) {
-      for (let i=0; i<behaviors.length; i++) {
-        let b = behaviors[i];
-        if (b) {
-          klass = Array.isArray(b) ? _mixinBehaviors(b, klass) :
-            GenerateClassFromInfo(b, klass);
+    function copyBehaviorProperties(behaviors, klass) {
+      const meta = {};
+      const superMeta = klass.prototype.__behaviorMetaProps;
```

Comment:
Since these are all lifecycle-related, maybe `__behaviorCallbacks` would be better than something with "props" in it, since that's sort of overloaded with all the data stuff.

Reply:
removed.